docs/index.html

```diff
@@ -0,0 +1,189 @@
+<!DOCTYPE HTML>
+<!--
+     Stellar by HTML5 UP
+     html5up.net | @ajlkn
+     Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
+-->
+<html>
+  <head>
+    <title>Blorg - The unicorn dairy</title>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+    <link rel="stylesheet" href="http://localhost:8080/static/css/main.css" />
```

Comment:
should this be a `url_for`?

Reply:
sorry I got confused. this is the generated HTML, not the source heheh! Sorry about that. we might have to do something like this to get the final URL here though, just like in the templatel docs: https://github.com/clarete/templatel/blob/master/docs/publish.el#L64-L66